build fail likely real
```
======================================================================
ERROR: test_basic_cloudant_seq (corehq.apps.hqadmin.tests.test_utils.TestPillowCheckpointSeqStore)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/hqadmin/tests/test_utils.py", line 25, in test_basic_cloudant_seq
    store = PillowCheckpointSeqStore.objects.get(checkpoint_id=self.pillow.checkpoint.checkpoint_id)
  File "/vendor/lib/python2.7/site-packages/django/db/models/manager.py", line 92, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/django/db/models/query.py", line 357, in get
    self.model._meta.object_name)
DoesNotExist: PillowCheckpointSeqStore matching query does not exist.
```

--- a/corehq/apps/hqadmin/tests/test_utils.py
+++ b/corehq/apps/hqadmin/tests/test_utils.py
@@ -1,4 +1,5 @@
 from django.test import TestCase, override_settings
+from mock import patch
 
 from ..models import PillowCheckpointSeqStore
 from ..utils import pillow_seq_store, EPSILON
@@ -16,7 +17,14 @@ class TestPillowCheckpointSeqStore(TestCase):
     dependent_apps = ['pillowtop']
 
     def setUp(self):
+        super(TestPillowCheckpointSeqStore, self).setUp()
         self.pillow = DummyPillow()
+        self.pillow_patch = patch("corehq.apps.hqadmin.utils.get_couch_pillow_instances", return_value=[DummyPillow()])
+        self.pillow_patch.start()
+
+    def tearDown(self):
+        self.pillow_patch.stop()
+        super(TestPillowCheckpointSeqStore, self).tearDown()
 
     def test_basic_cloudant_seq(self):
         seq = '1-blahblah'

